### PR TITLE
Set up Node.js and Express

### PR DESCRIPTION
**Pull request by deleonkf**
Changes:
- Set up Node.js and Express
- Set up a proxy at http://localhost:5000/
- Created GET endpoint to test connection between frontend and backend (will remove in future work)

### CODE REVIEW

**Comment by mariellemanlulu** [comment]
Hey faith, how do I test this? Like how do I set up a proxy?

**Comment by deleonkf** [comment]
> Hey faith, how do I test this? Like how do I set up a proxy?

Woops sorry! Pushing updates into the README for it to be more clear. You'll need to run this server in one terminal and the client in another.

File: web_app/text_savvy_backend/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+3. Install dependencies.
+    ```
+    $ npm install
+    ```
+    - Update `npm` if necessary.
+4. Run the server.
+    ```
+    $ npm start
+    ```
+5. Visit [localhost:5000](http://localhost:5000) in Chrome. Port 5000 is where the Express server will be running on.
```
Comment (mariellemanlulu) [inline web_app/text_savvy_backend/README.md:23]:
```suggestion
5. Visit [localhost:5000/api](http://localhost:5000/api) in Chrome. Port 5000 is where the Express server will be running on.
```

**Review by mariellemanlulu — APPROVED** [review]
just minor changes in read me and merge conflict (which i think shouldn't be too much of a problem) so i'm approving!

**Pull request merged**